docs/platforms/python/tracing/distributed-tracing/custom-instrumentation/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Custom Instrumentation
```

Comment:
@sfanahata - In the updated JavaScript tracing docs, we changed this section to be "Custom Trace Popagation" to make it clearer what someone was getting out of this doc. We should match that.